sd-whonix/app-journalist.yaml

```diff
@@ -0,0 +1 @@
+{{ hostname }}:descriptor:x25519:{{ key }}
```

Comment:
:heavy_check_mark: this will be removed everytime `clean-salt` is called via https://github.com/freedomofpress/securedrop-workstation/blob/master/Makefile#L56